app-modules/resource-hub/src/Filament/Resources/ResourceHubArticles/Pages/EditResourceHubArticle.php

```diff
@@ -178,6 +178,15 @@ protected function getSavedNotificationTitle(): ?string
     protected function getFormActions(): array
     {
         return [
+            BaseAction::make('cancel')
+                ->label(__('filament-panels::resources/pages/edit-record.form.actions.cancel.label'))
+                ->color('gray')
+                ->requiresConfirmation()
```

Comment:
Is there some Filament built in action we can use here? If not, `requiresConfirmation` should be conditional as to whether or not there actually are unsaved changes